core/iwasm/interpreter/wasm_loader.c

```diff
@@ -4933,6 +4959,64 @@ wasm_loader_ctx_reinit(WASMLoaderContext *ctx)
     return true;
 }
 
+static void
+wasm_loader_emit_float64(WASMLoaderContext *ctx, float64 value)
```

Comment:
How about merging `wasm_loader_emit_float64/float32/uint64` into a single function 

```C
wasm_loader_emit_const(WASMLoaderContext *ctx, void *value, bool is_32_bit)
{
        bh_memcpy_s(ctx->p_code_compiled,
                    ctx->p_code_compiled_end - ctx->p_code_compiled, &value,
                    is_32_bit ? sizeof(uint32) : sizeof(uint64));
        ctx->p_code_compiled += is_32_bit ? sizeof(uint32) : sizeof(uint64);
        ...
}
```

Reply:
Done, thanks.

I remain emit_uint32 unchanged since it has been used in several places.